src/store/storage-engine.js

```diff
@@ -18,6 +18,8 @@ function rejectWithMessage(error) {
 export default function createEngine(key: string) {
   const store = new ElectronStore();
 
+  window.electronStore = store;
```

Comment:
I meant to do this in #159, but must've forgotten. This makes it possible to clear the state through the developer tools.